test/ComparedQueryable.Test/packages.config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="FluentAssertions" version="4.19.2" targetFramework="net462" />
+  <package id="xunit" version="2.2.0" targetFramework="net462" />

```

Comment:
xunit seemed like a nice testing framework to use. Plus it seemed compatible with .NET core projects.